src/main/java/com/tripply/Auth/service/Impl/AuthServiceImpl.java

```diff
@@ -86,6 +108,63 @@ public ResponseModel<String> blockToken(String jwt) {
         log.info("AuthService: blockToken() ended with jwt -> {}", jwt);
         return response;
     }
+    @Override
+    public ResponseModel<String> forgotPassword(String email) {
+        log.info("AuthService : forgotPassword() started with email -> {}" , email);
+        ResponseModel<String> response = new ResponseModel<>();
+        Optional<User> user = userRepository.findByEmail(email);
+        if (user.isEmpty()){
+            throw new RecordNotFoundException("User Not Found");
+        }
+        PasswordResetToken passwordResetToken = new PasswordResetToken();
+        passwordResetToken.setUser(user.get());
+        passwordResetToken.setToken(RandomTokenGenerator.generateToken());
+        passwordResetTokenRepository.save(passwordResetToken);
+
+        try {
+            return webClientService.postWithParameterizedTypeReference(notificationBaseUrl + SEND_FORGET_PASSWORD_EMAIL,
```

Comment:
This API should be whiltelist in notification also. make sure that

Reply:
Resolved